rhel6/profiles/standard.profile

```diff
@@ -190,7 +190,7 @@ selections:
     - mount_option_noexec_removable_partitions
     - require_smb_client_signing
     - mount_option_smb_client_signing
-    - accounts_password_pam_unix_remember
+    - accounts_passwords_pam_unix_remember
```

Comment:
These are backwards incompatible changes, and affect many users with tailored profiles. Please don't make these type changes.